youtube_dl/extractor/nexx.py

```diff
@@ -69,28 +69,14 @@ class NexxIE(InfoExtractor):
         'info_dict': {
             'id': '1269984',
             'ext': 'mp4',
-            'title': '1 TAG ohne KLO... wortwörtlich! 😑',
-            'alt_title': '1 TAG ohne KLO... wortwörtlich! 😑',
-            'thumbnail': r're:^https?://.*\.jpg$',
+            'title': '1 TAG ohne KLO... wortwörtlich! ?',
+            'alt_title': '1 TAG ohne KLO... wortwörtlich! ?',
+            'description': 'md5:1cf562de6653f76fbc589ecbdf985727',
+            'thumbnail': r're:^https?://.*\.(?:jpg|webp)$',
             'duration': 607,
             'timestamp': 1518614955,
             'upload_date': '20180214',
         },
-    }, {
-        # free cdn from http://www.spiegel.de/video/eifel-zoo-aufregung-um-ausgebrochene-raubtiere-video-99018031.html
-        'url': 'nexx:747:1533779',
```

Comment:
I moved this test to the spiegel.de extractor (future PR), is that acceptable?

Reply:
No. Spiegel does not match `nexx:`.

Reply:
Well I obviously changed the url. They appear to no longer use nexx anyway...